autopush/db.py

```diff
@@ -436,7 +436,7 @@ def save_channels(self, uaid, channels):
 
     @track_provisioned
     def store_message(self, notification):
-        # type: (WebPushNotification) -> bool
+        # type: (WebPushNotification)
```

Comment:
mypy wants -> None here (don't ask me why because I do this all the time)

Reply:
Oh, huh, and pycharm noted that. Need to make it a more important warning.